components/esp_lvgl_port/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "2.3.0"
+version: "2.3.0~1"
```

Comment:
Could we keep the version for now? I expect more changes to esp_lvgl_port soon

Reply:
ok